packages/core/src/storage/StorageUtility.spec.ts

```diff
@@ -444,33 +445,19 @@ describe("StorageUtility", () => {
   });
 });
 
-describe("loadOidcContextFromStorage", () => {
-  it("throws if no stored state matches the current request's", async () => {
-    const mockedStorage = mockStorageUtility({
-      "solidClientAuthenticationUser:mySession": {
-        issuer: "https://my.idp/",
-        codeVerifier: "some code verifier",
-        redirectUri: "https://my.app/redirect",
-        dpop: "true",
-      },
-    });
+describe("getSessionIdFromOauthState", () => {
+  it("returns undefined if no stored state matches the current request's", async () => {
+    const mockedStorage = mockStorageUtility({});
 
     await expect(
-      loadOidcContextFromStorage(
-        "some unexisting state",
-        mockedStorage,
-        mockIssuerConfigFetcher(mockIssuerConfig())
-      )
-    ).rejects.toThrow(
-      "Failed to retrieve OIDC context from storage for login request associated with state"
-    );
+      getSessionIdFromOauthState(mockedStorage, "some unexisting state")
```

Comment:
```suggestion
      getSessionIdFromOauthState(mockedStorage, "some non-existent 'state' value")
```

Reply:
Resolved in 0ba475eb12e5452ea5a8427d6f1417b422d7fc4f